### PR TITLE
Correct the date time format for deletedTime

### PR DESCRIPTION
**Pull request by zfchen95**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by blueww** [comment]
I don't see the .net SDK result. If it's pass, I am OK with it.

**Comment by chiragg4u** [comment]
Please go through the contribution checklist added as part of the review and mark them completed.

**Review by blueww — APPROVED** [review]
As .net SDK pass, I am OK with it from .net SDK side.
Anyway, change the type of existing property might be breaking, it need ARM team board review.

**Comment by Juliehzl** [comment]
python SDK works for azure cli. @zfchen95 would you add such change in container in this PR?

**Comment by zfchen95** [comment]
> python SDK works for azure cli. @zfchen95 would you add such change in container in this PR?

We will do the change in server for container soft delete. The existing format in swagger is good.

File: specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json [hunk 1]
```diff
@@ -673,7 +673,7 @@
         },
         "deletedTime": {
           "type": "string",
-          "format": "date-time",
+          "format": "date-time-rfc1123",
```
Comment (chiragg4u) [inline specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json:676]: Will this break existing SDK's and usage by the customers?

Comment (blueww) [reply specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json:676]: As the .net SDK test pass, should be good to .net SDK.
But I see Go SDK test fail, The Go SDK owner might need to look if it caused go SDK breaking.

**Comment by ravbhatnagar** [comment]
looks like a breaking change. also, i am not sure if the new format is supported. we should be consistent within all azure services here. and as far as I know, format: date-time is what we support. So, please confirm with SDK folks. @johanste - any comment on this?

**Comment by johanste** [comment]
Is this a change in service behavior? If so, it should not be made since it is a (semantic) breaking change. There are absolutely no guarantees that a given client understands RFC822 date times just because it can parse ISO 8601 formats.

While rfc1123 is a known format to autorest, it is an extension, and most clients/code generators would not be aware.

**Pull request closed without merging**